classes/Product.php

```diff
@@ -2233,7 +2233,7 @@ public function updateAttribute(
         array $id_shop_list = [],
         $isbn = '',
         $low_stock_threshold = null,
-        $low_stock_alert = false,
+        ?bool $low_stock_alert = false,
```

Comment:
low_stock_alert can't be NULL. Database structure doesn't allow NULL for low_stock_alert:
`low_stock_alert	tinyint(1) [0]`

Reply:
Good catch - https://github.com/PrestaShop/PrestaShop/blob/9781b9ed82627de5c2cd5bf3af77259d69e345d3/install-dev/data/db_structure.sql#L1747

Reply:
This is true but into the method as you can see:

`'low_stock_alert' => null !== $low_stock_alert,`

this is used to know is a field must bu updated or not. Indeed at line 2322 the field is never setted as null because we use:
` $combination->low_stock_alert = !empty($low_stock_alert);`